packages/core/lib/combo/patch-build/randomizer.ts

```diff
@@ -761,6 +761,9 @@ const randomizerDungeonsBits = (worldId: number, logic: LogicResult): Buffer =>
 
   for (let i = 0; i < DUNGEONS_PRECOMPLETED.length; ++i) {
     const dungeon = DUNGEONS_PRECOMPLETED[i];
+    if (settings.erPiratesWorld && world.preCompleted.has('PF')) {
```

Comment:
This will never be true as PF cannot be a pre-completed dungeon.

Reply:
Well, PF is part of the pre-completed dungeon list, so I'm not quite sure what you mean by that.

Reply:
Check the [`selectPreCompletedDungeons`](https://github.com/OoTMM/OoTMM/blob/master/packages/core/lib/combo/logic/solve.ts#L810-L818) function and the other functions it calls. Only Major Dungeons can be pre-completed.